CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+0.6.1 / 2022-09-30
+===================
+
+### Features
+* Implemented filtering of reports by annotation (#121)
+* Minimal and default agent versions upgraded to `1.0.682`
+
+### Bugfixes
+* Added filtering out projects without a build file (#222)
+* Added JaCoCo reports filtering (#220)
+* Fixed coverage for function reference (#148)
+* Fixed incorrect multiplatform lookup adapter (#193)
+
+### Internal features
```

Comment:
Are internal features really needed in the changelog?

Reply:
I think this is not necessary, but it can be useful for contributors, it can also be useful in development to study all changes between versions without analyzing the git log.